kernel/arch/dreamcast/include/dc/asic.h

```diff
@@ -193,9 +195,17 @@ typedef void (*asic_evt_handler)(uint32_t code);
 
     \param  code            The ASIC event code to hook (see \ref asic_events).
     \param  handler         The function to call when the event happens.
+    \param  data            A user pointer that will be passed to the callback.
 
 */
-void asic_evt_set_handler(uint16_t code, asic_evt_handler handler);
+void asic_evt_set_handler(uint16_t code, asic_evt_handler handler, void *data);
+
```

Comment:
These two would need to get fleshed out dox.

Reply:
What do you mean? It's already documented.

Reply:
Nevermind - just noticed you were talking about the two functions below this one.

Reply:
> Nevermind - just noticed you were talking about the two functions below this one.

Right, sorry, it made more sense in the review view where these are all inlined so the comment would have appeared directly in the spot where the dox go. I forgot they get truncated down in the normal views -_-;